tests/bootstrap.php

```diff
@@ -0,0 +1,22 @@
+<?php
+/**
+ * This file is loaded by PHPUnit before any test.
+ */
+
+use PHPUnit\DbUnit\DataSet\YamlDataSet;
+use PHPUnit\DbUnit\TestCaseTrait;
+use PHPUnit\Framework\TestCase;
+
+require_once __DIR__.'/../boot.php';
+require_once __DIR__.'/../include/api.php';
+
+// Backward compatibility
+if (!class_exists(TestCase::class)) {
```

Comment:
Is that really okay for PHP 5.6 ? This is the current lowest version Friendica is supporting until all major distris have dumped it. And maybe you want then `' if (!class_exists('TestCase')) {` and so on.

Reply:
Yes, Travis is running these tests with PHP 5.6 among others.

Reply:
Yes, 5.6 does not have the TestCase class, so the class_exists will return false and instead the alias for the class will be set.

Reply:
And `::class` was added in PHP 5.5 so this code should work everywhere.

Reply:
Okay, I did not know it.